.github/workflows/deploy-docker.yaml

```diff
@@ -7,6 +7,7 @@ name: Build Docker image & push
 on:
   release:
     types: [published]
+  pull_request:
```

Comment:
do not forget to remove this

Reply:
Should be fine to keep it, as it doesn't push the images on pull request:
`push: ${{ github.event_name != 'pull_request' }}`